src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3990,7 +4008,7 @@ private void MarkDependentSlotsNotNull(int slot, TypeSymbol expressionType, ref
                     member.RequiresInstanceReceiver() &&
                     (containingType is null || AccessCheck.IsSymbolAccessible(member, containingType, ref discardedUseSiteDiagnostics)))
                 {
-                    int childSlot = GetOrCreateSlot(member, slot, true);
+                    int childSlot = GetOrCreateSlot(member, slot, forceSlotEvenIfEmpty: true, createIfMissing: false);
```

Comment:
>true [](start = 88, length = 4)

It feels inconsistent passing true here and false for the next argument

Reply:
I'll address in a subsequent PR.